### DISCUSSION TITLE
How to perform openthread test authentication

### DISCUSSION THREAD

**Discussion by Evan05300** [openthread/openthread#8294] [Q&A] [answered]
Excuse me,
but how can I download Harness Thread Test Solftware? I can't find the download address

**Comment by jwhui** [comment, marked as answer]
The Thread Group Test Harness is available to [Thread Group](https://www.threadgroup.org/) members. Please contact the [Thread Group Help Desk](mailto:help@threadgroup.org) with any questions.
